### PR TITLE
Time select

### PR DESCRIPTION
**Pull request by rahulssheth1**

### CODE REVIEW

**Review by haowie-dev — COMMENTED** [review]
Next time use a dash instead of camel case for branch naming

**Comment by haowie-dev** [comment]
Can we make it so the user can see the time they set in the view after setting it? Other than that its gucci - move fast break things lmao

File: src/screens/TimeSelectScreen.js [hunk 1]
```diff
@@ -35,19 +40,65 @@ export default class TimeSelectScreen extends React.Component {
             (auth) => this.setState({ auth: auth })
         )
     }
+    showStartDateTimePicker = () =>
+        this.setState({
```
Comment (michaelnyu) [inline src/screens/TimeSelectScreen.js:44]: might be better to use Object.assign (this.setState())

File: src/screens/TimeSelectScreen.js [hunk 2]
```diff
@@ -35,19 +40,65 @@ export default class TimeSelectScreen extends React.Component {
...
         )
     }
+    showStartDateTimePicker = () =>
+        this.setState({ 
+            isDateTimePickerVisible: true,
+            startTimeChosen: true, 
+        });
+
+    showEndDateTimePicker = () => 
+        this.setState({
```
Comment (michaelnyu) [inline src/screens/TimeSelectScreen.js:50]: same as above

File: src/screens/TimeSelectScreen.js [hunk 3]
```diff
@@ -35,19 +40,65 @@ export default class TimeSelectScreen extends React.Component {
...
+            isDateTimePickerVisible: true,
+            endTimeChosen: true,
+        })
+    
+    hideDateTimePicker = () =>
+        this.setState({
```
Comment (michaelnyu) [inline src/screens/TimeSelectScreen.js:56]: same as above

File: src/screens/TimeSelectScreen.js [hunk 4]
```diff
@@ -35,19 +40,65 @@ export default class TimeSelectScreen extends React.Component {
...
+    hideDateTimePicker = () =>
+        this.setState({ 
+            isDateTimePickerVisible: false,
+            startTimeChosen: false,
+            endTimeChosen: false, 
+
+        }); 
+
+    handleTimePicked = (date) =>{
+        if (this.state.startTimeChosen)
```
Comment (michaelnyu) [inline src/screens/TimeSelectScreen.js:64]: same as above

File: src/screens/InviteScreen.js [hunk 5]
```diff
@@ -45,13 +46,15 @@ export default class InviteScreen extends React.Component {
...
-		const curMoment = moment(this.props.navigation.state.params.startTime).format("MM/DD/YYYY")
+		const startMoment = moment(this.props.navigation.state.params.startTime).format("MM/DD/YYYY")
+		const endMoment = moment(this.props.navigation.state.params.endTime).format("MM/DD/YYYY")
 		const { navigate } = this.props.navigation
 
 		return (
 			<View style={styles.container} >
 				<Text style={styles.creationSubText}> Time</Text>
-				<Text style={styles.creationSubText}> {curMoment} </Text>
+				<Text style={styles.creationSubText}> {startMoment} </Text>
```
Comment (michaelnyu) [inline src/screens/InviteScreen.js:56]: remove space before and after

File: src/screens/InviteScreen.js [hunk 6]
```diff
@@ -13,6 +13,7 @@ import {
 	ListItem,
 } from 'react-native-elements'
 
+
 import storage from '../globals/storage'
```
Comment (michaelnyu) [inline src/screens/InviteScreen.js:17]: should we use semicolons for everything? i dont really know what is best practice

File: src/screens/HomeScreen.js [hunk 7]
```diff
@@ -63,7 +63,9 @@ export default class HomeScreen extends React.Component {
 				`&client_id=${FB_APP_ID}` +
 				`&redirect_uri=${encodeURIComponent(redirectUrl)}`,
 		})
-
+		if (result.type == 'error') {
+		    console.log(result.errorCode)
```
Comment (michaelnyu) [inline src/screens/HomeScreen.js:67]: handle error another way. maybe make a conditional render based on state.

this.state = {
   ...,
   err = false,
}
....

render() {
   {err && < Text >{err}</ Text >}
}

on error change this.state.err = "result.errorCode" or something.

**Review by michaelnyu — CHANGES_REQUESTED** [review]

**Pull request merged**